tools/server/webui/src/Config.ts

```diff
@@ -41,6 +43,14 @@ export const CONFIG_DEFAULT = {
   custom: '', // custom json-stringified object
   // experimental features
   pyIntepreterEnabled: false,
+  // Fields for tool calling
+  streamResponse: true,
+  ...Object.fromEntries(
+    Array.from(AVAILABLE_TOOLS.values()).map((tool: AgentTool) => [
+      `tool_${tool.id}_enabled`,
+      false, // Default value for tool enabled state (e.g., false for opt-in)
+    ])
+  ),
```

Comment:
I think this can be kept simple for now, by simply listing the list of tools here (we don't have many, right?)

Reply:
Well, sure can, but I think we will need to readd it in the future?
https://github.com/ggml-org/llama.cpp/pull/13501#issuecomment-2895166079

Reply:
Ok!